### ISSUE TITLE
Refreshing page results in 404

### ISSUE THREAD

**Issue by tesar-tech** [mizrael/BlazorOnGitHubPages#2]
Thanks for article and this repo. 

I have just one issue I could not solve yet. Refreshing page in others locations than index (e.g.: https://mizrael.github.io/BlazorOnGitHubPages/counter) throws 404. It also doesn't work if you click that link. 

In this particular example it changes the url to https://mizrael.github.io/?p=/BlazorOnGitHubPages/counter  (with `?p=` inside). But also without this mistake it doesn't work either (tested here: https://tesar-tech.github.io/blazorTestWithGhPages/counter)...

It is probably connected to `<base` tag with changed `href`...

Any ideas here?

**Comment by mizrael** [comment]
hi! Sorry for the late reply, I just realized there was a notification pending! Did you manage to fix it?

**Comment by tesar-tech** [comment]
No, but now I know why is it happening. The server which serves WA Blazor app doesn't know what to serve on alternative urls (like /counter). More information [here](https://docs.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-5.0#rewrite-urls-for-correct-routing).

**Comment by mizrael** [comment]
I'll take a look today/tomorrow and get back to you ASAP

Il giorno lun 23 nov 2020 alle ore 12:12 Jan Tesař <notifications@github.com>
ha scritto:
